mixpanel_query/client.py

```diff
@@ -871,7 +871,9 @@ def get_export(self, start_date, end_date, event=None, where=None, bucket_id=Non
         if start_date_obj > end_date_obj:
             raise exceptions.InvalidDateException('The `start_date` specified after the `end_date`; you will not receive any events.')
 
-        if isinstance(event, str):
+        # the provided event should be an array even when a singleton
+        # if a singleton string/unicode is provided, put it into an array
+        if event is not None and not isinstance(event, list):
```

Comment:
You can actually use `isinstance(event, basestring)` if you want. This would be `True` for a `str` and a `unicode` object.